packages/gatsby-transformer-sqip/src/extend-node-type.js

```diff
@@ -1,22 +1,23 @@
-const { resolve } = require(`path`)
-const md5File = require(`md5-file`)
-
-const {
-  DuotoneGradientType,
-  ImageCropFocusType,
-} = require(`gatsby-transformer-sharp/types`)
-const { queueImageResizing } = require(`gatsby-plugin-sharp`)
+const path = require(`path`)
 
 const Debug = require(`debug`)
 const fs = require(`fs-extra`)
+const sharp = require(`sharp`)
+const md5File = require(`md5-file`)
+
 const {
   GraphQLObjectType,
   GraphQLString,
   GraphQLInt,
   GraphQLBoolean,
 } = require(`gatsby/graphql`)
-const sharp = require(`sharp`)
-const { ensureDir } = require(`fs-extra`)
+const { queueImageResizing } = require(`gatsby-plugin-sharp`)
+const { fetchRemoteFile } = require(`gatsby-core-utils`)
+const { createUrl } = require(`gatsby-source-contentful/extend-node-type`)
```

Comment:
We won't want to couple this to the contentful plugin so we should copy the code into this plugin.

Reply:
Which code do you mean by `this`? :)

Reply:
I mean createUrl, we don't want gatsby-transformer-sqip to have a dependency on gatsby-source-contentful

Reply:
Moved the require within the resolver which is only available when you got gatsby-source-contentful enabled :)

Reply:
I didn't know we had `SqipContentful` in here, that's definitely a design flaw on our end.